morebits.js

```diff
@@ -2930,36 +2930,72 @@ Morebits.wiki.page = function(pageName, status) {
 	};
 
 	/**
+	 * Set whether and how to watch the page, including setting an expiry.
+	 *
 	 * @param {boolean|string} [watchlistOption=false] -
 	 * Basically a mix of MW API and Twinkley options available pre-expiry:
-	 * - `true`|`'yes'`: page will be added to the user's watchlist when the action is called
-	 * - `false`|`'no'`|`'nochange'`: watchlist status of the page will not be changed.
-	 * - `'default'`|`'preferences'`: watchlist status of the page will
-	 * be set based on the user's preference settings when the action is
-	 * called.  Ignores ability of default + expiry.
-	 * - `'unwatch'`: explicitly unwatch the page
-	 * - {string|number}: watch page until the specified time (relative or absolute datestring)
+	 * - `true`|`'yes'`|`'watch'`: page will be added to the user's
+	 * watchlist when the action is called. Defaults to an indefinite
+	 * watch unless `watchlistExpiry` is provided.
+	 * - `false`|`'no'`|`'nochange'`: watchlist status of the page (including expiry) will not be changed.
+	 * - `'default'`|`'preferences'`: watchlist status of the page will be
+	 * set based on the user's preference settings when the action is
+	 * called. Defaults to an indefinite watch unless `watchlistExpiry` is
+	 * provided.
+	 * - `'unwatch'`: explicitly unwatch the page.
+	 * - Any other `string` or `number`: watch page until the specified
+	 * time, deferring to `watchlistExpiry` if provided.
+	 * @param {string|number} [watchlistExpiry=infinity] - A date-like string or number.
```

Comment:
What number? The switch doesn't seem to cover the case of a number.

Reply:
Actually wouldn't it make sense to allow a Date or Morebits.date object here (to be passed to API after `.toISOString`ing it)?

Reply:
> Actually wouldn't it make sense to allow a Date or Morebits.date object here (to be passed to API after `.toISOString`ing it)?

Oh that'd be neat, for all involved!